### DISCUSSION TITLE
This app is optimized for televisions

### DISCUSSION THREAD

**Discussion by ACR-Jeff** [jellyfin/jellyfin-androidtv#2017] [General]
Every time I open Jellyfin-Androidtv version 0.14.1 it pops up a notice stating "This app is optimized for televisions. We recommend using our mobile app on other devices" 
I am using Android TV. What I am wanting to know is there any way to stop this message from appearing?

**Comment by nielsvanvelzen** [comment]
This notification should only show for non-tv devices. Please create an issue with all asked info (device model, android version etc) so we can look into it.

**Comment by EatedPantsu** [comment]
Can this be removed? I modified old android phone to be a android TV box of a sort, and it works excellent except this irritating windows. Can it be permanently removed?

**Reply by nielsvanvelzen** [reply]
You cannot permanently dismiss the notification right now, please open an issue so we can look into it.
